.circleci/config.yml

```diff
@@ -60,3 +71,14 @@ workflows:
           filters:
             branches:
               only: master
+      - release:
+          context: falco
+          requires:
+            - test
+            - lint
+            - build-image
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*(-.*)*/
```

Comment:
Until now, I never prefixed release version with `v`, do we force the usage or add `v` as optionnal in regexp?

Reply:
usually is good to prefix, but if we are not using that I will remove

Reply:
ptal